third_party/accessibility/ax/platform/ax_platform_node_mac.mm

```diff
@@ -834,8 +834,7 @@ - (BOOL)isAccessibilitySelectorAllowed:(SEL)selector {
   // accessibility clients from trying to set the selection range, which won't
   // work because of 692362.
   if (selector == @selector(setAccessibilitySelectedText:) ||
-      selector == @selector(setAccessibilitySelectedTextRange:) ||
-      selector == @selector(setAccessibilitySelectedTextMarkerRange:)) {
```

Comment:
Note: this selector does not exist and generates a compile error. The [original code](https://source.chromium.org/chromium/chromium/src/+/main:ui/accessibility/platform/ax_platform_node_cocoa.mm) has been rewritten since we forked it and this line no longer exists there either.

The use of "MarkerRange" instead of "MarkRange" is suspicious but neither is valid API and the only results for this selector in google are us.